acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -113,7 +113,21 @@ const verbRedirMap = {
   'number-pages': 'number',
 };
 
+const exhCreateLimitCookie = 'cr_p_c_e';
+const exhExportLimitCookie = 'ex_p_c_e';
+const exhOrganizeLimitCookie = 'or_p_c_e';
+const exhCompressLimitCookie = 'cm_p_ops_e';
+
+const exhLimitCookieMap = {
+  'to-pdf': exhCreateLimitCookie,
+  'pdf-to': exhExportLimitCookie,
+  'compress-pdf': exhCompressLimitCookie,
+  'rotate-pages': exhOrganizeLimitCookie,
+};
```

Comment:
probably ok to inline the cookie strings from L116-119

Reply:
Updated